commands/adminsonline.js

```diff
@@ -11,9 +11,30 @@ module.exports = {
         const rconpw = process.env.RCONPASS;
         const rconToSend = `/sc local admins, ctn = {}, 0 for _, p in ipairs(game.connected_players) do if p.admin then ctn = ctn + 1 admins[ctn] = p.name end end rcon.print('Online: '..ctn..': '..table.concat(admins, ', '))`; //get Admins, then add count, then display.
         const server = args[0];
```

Comment:
this shood now be 
```suggestion
        const server = Number(args[0]) ||args[0]|;
``` 
as you are  using it as a number

Reply:
but all is not a number, at some point I think I would like to use the po features in admin.

Reply:
that why we have const server = Number(args[0]) ||args[0]|; so if number does not resolve (its all) it will be all

Reply:
ok

Reply:
`const  server = Number(args[0]) ||args[0]|; `
is the same as 
`if(Number(args[0])){
    server = Number(args[0])
}else{
server = args[0]
}`